apis/openstack-cinder/src/test/java/org/jclouds/openstack/cinder/v2/internal/BaseCinderApiLiveTest.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jclouds.openstack.cinder.v2.internal;
+
+import org.jclouds.apis.BaseApiLiveTest;
+import org.jclouds.openstack.cinder.v2.CinderApi;
+import org.jclouds.openstack.keystone.v2_0.config.KeystoneProperties;
+
+import java.util.Properties;
+
+/**
+ * Tests behavior of CinderV2Api
+ */
+public class BaseCinderApiLiveTest extends BaseApiLiveTest<CinderApi> {
```

Comment:
you don't need this class you need to use org.jclouds.openstack.cinder.v1.internal.BaseCinderApiLiveTest